client/tests/sample.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
we are probably going go back to testing and write more elaborate test so we can do this later. But in general I think it's good to test for both positives and negatives